docs/source/teams/pluggable_auth.md

```diff
@@ -0,0 +1,683 @@
+    "clientSecret": "...",
+    "issuer": "https://keycloak.dev.acme.ai/auth/realms/acme",
+    "token": {
+        "url": "https://keycloak.dev.acme.ai/auth/realms/acme/protocol/openid-connect/token"
+    },
+    "userinfo": {
+        "url": "https://keycloak.dev.acme.ai/auth/realms/acme/protocol/openid-connect/userinfo"
+    },
+    "idToken": true,
+    "style": {
+        "logoDark": "https://images.com/keycloak.svg",
+        "bg": "#47abc6",
+        "text": "#fff"
+    },
+    "profile": "(e,l)=>(console.log({profile:e,tokens:l}),{id:e.sub,name:e.name,email:e.email,image:null})",
+    "allowDangerousEmailAccountLinking": true
+}
+```
+
+<!--
+**Migrating from Legacy to internal mode**
+
+The migration from Legacy to internal mode begins with configuring an
+authentication provider through CAS (UI or REST). Following this, all existing
+user data must be migrated from Auth0, utilizing the management SDK or Auth0
+API to ensure a comprehensive transfer of information.
+
+For each user, the migration involves several steps:
+
+1. Creating a FiftyOne Teams user profile via `POST /cas/api/users`.
+1. Assigning the user to the default organization through a membership entry.
+1. Linking the user's account to the new authentication provider by creating an
+   account reference via `POST /cas/api/accounts`.
+
+The final step in the migration involves changing the `FIFTYONE_AUTH_MODE`
+setting from legacy to internal. This change officially activates Internal
+Mode, completing the migration process.
+-->
+
+## Getting Started with Internal Mode
+
+These steps are only required to run FiftyOne Teams in
+[internal mode](#internal-mode). Please skip when using Auth0 in
+[legacy mode](#legacy-mode).
+
+1. Configure your Identity Provider
+1. Login to the SuperUser UI by navigating to
+   `https://≤YOUR_FIFTYONE_TEAMS_URL>/cas/configurations` and in the top right,
+   and provide the `FIFTYONE_AUTH_SECRET` to login.
+1. Create an Admin
+    1. Click on the “Admins” tab
+    1. Click “Add admin” in the bottom left
+    1. Set the name and email address (as it appears in your Identity Provider)
+       and click “Add”
+1. Add your Identity Provider
+    1. Click on the “Identity Providers” tab at the top of the screen and click
+       “Add provider”
+    1. Fill out the “Add identity provider”
+        1. You can also click “Switch to advanced editor” to provide the full
+           configuration as a JSON object
+    1. In the “Profile callback” field, set the mapping that your Identity
+       Provider expects
+1. Login with the admin user
+    1. Navigate to `https://≤YOUR_FIFTYONE_TEAMS_URL>/datasets`
+    1. You should see the login screen for your newly configured authentication
+       provider
+    1. Before logging in, set the admin user (in step 5). Otherwise, you will
+       need to remove this user from the database and try again.
+    1. Click the login button and provide the credentials of the Admin user
+       (set in step 5)
```

Comment:
Here should the "in step 5" be "in step 3"

Reply:
```suggestion
       (set in step 3)
```